config/config.exs

```diff
@@ -65,6 +52,16 @@ config :hex_web, HexWeb.Endpoint,
   secret_key_base: "Cc2cUvbm9x/uPD01xnKmpmU93mgZuht5cTejKf/Z2x0MmfqE1ZgHJ1/hSZwd8u4L",
   render_errors: [accepts: ~w(html json elixir erlang)]
 
+config :hex_web, HexWeb.Mailer,
+  adapter: Bamboo.SMTPAdapter,
+  server: System.get_env("HEX_SES_ENDPOINT") || "email-smtp.us-west-2.amazonaws.com",
+  port: System.get_env("HEX_SES_PORT") || 587,
```

Comment:
That's a good point - but this is the place we pass it to Bamboo, so we need to make sure it's an integer here.